The manpage for Screen says that you should use C-a C-\ to quit screen, while
it is in fact C-a \ (Closes: #630535)

Index: screen/doc/screen.1
===================================================================
--- screen.orig/doc/screen.1	2011-10-05 02:54:58.000000000 +0200
+++ screen/doc/screen.1	2011-10-05 02:55:51.000000000 +0200
@@ -584,7 +584,7 @@
 Write out a \*Q.termcap\*U file.
 .IP "\fBC-a ?\fP	(help)"
 Show key bindings.
-.IP "\fBC-a C-\e\fP	(quit)"
+.IP "\fBC-a \e\fP	(quit)"
 Kill all windows and terminate
 .IR screen .
 .IP "\fBC-a :\fP	(colon)"
Index: screen/doc/screen.texinfo
===================================================================
--- screen.orig/doc/screen.texinfo	2011-10-05 02:56:47.000000000 +0200
+++ screen/doc/screen.texinfo	2011-10-05 02:57:40.000000000 +0200
@@ -791,7 +791,7 @@
 (help)@*
 Show key bindings.  @xref{Help}.
 
-@item @kbd{C-a C-\}
+@item @kbd{C-a \}
 (quit)@*
 Kill all windows and terminate @code{screen}.  @xref{Quit}.
 
@@ -2031,13 +2031,10 @@
 
 @node Quit,  , Suspend, Session Management
 @section Quit
-@kindex C-\
+@kindex \
 @deffn Command quit
-(@kbd{C-a C-\})@*
-Kill all windows and terminate @code{screen}.  Note that on VT100-style
-terminals the keys @kbd{C-4} and @kbd{C-\} are identical.  So be careful
-not to type @kbd{C-a C-4} when selecting window no. 4.  Use the empty
-bind command (as in @code{bind "^\"}) to remove a key binding
+(@kbd{C-a \})@*
+Kill all windows and terminate @code{screen}.
 (@pxref{Key Binding}).
 @end deffn
 
